src/plugins/vis_builder/public/application/components/data_tab/utils/get_available_fields.ts

```diff
@@ -17,7 +17,7 @@ export const getAvailableFields = (
   filterFieldTypes: FieldTypes = '*'
 ) => {
   const filteredFields = fields.filter((field: IndexPatternField) => {
-    if (!field.aggregatable || isNestedField(field) || field.scripted) {
+    if (!field.aggregatable || isNestedField(field)) {
```

Comment:
Just a question: should this be behind a flag? 

I assume not as I think most customers would expect these to be there, but just checking if we need to manage this change for customers.

Reply:
@virajsanghvi  In my opinion we don't need a flag here, since this experience is going to be common for all the users and not configurable via flags or settings. This behavior is consistent with the current Discover experience.